.github/kokoro/continuous-install-200t.cfg

```diff
@@ -11,7 +11,7 @@ timeout_mins: 4320
 action {
   define_artifacts {
     # File types
-    regex: "**/symbiflow-arch-defs-install-200t*.tar.xz"
+    regex: "**/symbiflow-arch-defs-*.tar.xz"
```

Comment:
Don't see a change to the template python script?

Reply:
Done.

I am thinking whether we need the install-200t CI as the 200T is now built and packaged in the install one.